src/components/ProcessDocumentScreen/ProcessDocumentTitle/ProcessDocumentTitle.tsx

```diff
@@ -26,7 +26,7 @@ export const ProcessDocumentTitle: FunctionComponent<ProcessDocumentTitle> = ({
       case QueueState.PENDING:
         return (
           <>
-            <LoaderSpinner className="mr-2" width="24px" primary="#3B8CC5" />
+            <LoaderSpinner className="mr-2" width="24px" primary="#2D5FAA" />
```

Comment:
`primary` perhaps to put in `constants` folder and export from there

Reply:
ok